recipes-support/libunwind/libunwind_1.1.bbappend

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Add necessary patches according to:
+# https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md#build-for-armlinux
+
+SRC_URI_append += "\
+           file://libunwind.git-770152268807e460184b4152e23aba9c86601090.patch \
+           file://expand-arm-Validate-memory-before-access.patch \
+"
```

Comment:
Yeah, we can do it optional, but since you don't have any branches targeting krogoth I think we can just leave it out.
My original idea was that it didn't do any harm and only applied if you had 1.1 in the first place, but it did do harm. :-)